src/pages/Account/Admin.tsx

```diff
@@ -73,49 +77,84 @@ const VenueDetails: React.FC<VenueDetailsProps> = ({ venueId }) => {
 
   return (
     <>
-      <ul>
+      <div className="page-container-adminpanel-tabs">
         {[
           { url: `${match.url}`, label: "Venue Infos" },
-          { url: `${match.url}/appearance`, label: "Appearance" },
+          // { url: `${match.url}/appearance`, label: "Appearance" },
           { url: `${match.url}/events`, label: "Events" },
         ].map((tab) => (
-          <li
+          <div
             key={tab.url}
-            style={location.pathname === tab.url ? { color: "red" } : {}}
+            className={`page-container-adminpanel-tab ${
+              location.pathname === tab.url && "selected"
```

Comment:
usually I write sth like `${condition ? ... : ""}` otherwise it writes `false` in the className.